src/main/kotlin/org/rust/cargo/runconfig/command/CargoCommandConfiguration.kt

```diff
@@ -63,6 +63,17 @@ open class CargoCommandConfiguration(
 
     private var isRedirectInput: Boolean = false
     private var redirectInputPath: String? = null
+    private val redirectInputFile: File?
+        get() {
+            if (!isRedirectInput) return null
+            if (redirectInputPath?.isNotEmpty() != true) return null
+            val redirectInputPath = FileUtil.toSystemDependentName(ProgramParametersUtil.expandPathAndMacros(redirectInputPath, null, project))
+            var file = File(redirectInputPath)
+            if (!file.isAbsolute && workingDirectory != null) {
+                file = File(File(workingDirectory.toString()), redirectInputPath)
```

Comment:
This case is not checked by tests because `org.rustSlowTests.cargo.runconfig.RunConfigurationTest#test redirect input` pass absolute path
Let's add a test with relative path as well

Reply:
@mchernyavsky what about test with relative path?

Reply:
Oops, I haven't committed it.

Pushed in master - https://github.com/intellij-rust/intellij-rust/commit/13cecdb987f62c175937bd01d71fcffb8704e4fc